apps/src/templates/progress/ProgressLessonTeacherInfo.jsx

```diff
@@ -168,7 +168,7 @@ export default connect(
   state => ({
     section:
       state.teacherSections.sections[state.teacherSections.selectedSectionId],
-    unitAllowsHiddenLessons: state.hiddenLesson.hideableLessonsAllowed,
+    unitAllowsHiddenLessons: state.hiddenLesson.hideableLessonsAllowed || false,
```

Comment:
I could not figure out how to get the hiddenLessonRedux set up correctly because of the way it uses Immutable. If anyone has any ideas I'm totally open to trying something. I figured the other options were to add a default value here or to make the prop not be required. Adding this default seemed safe.